src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -13,15 +13,21 @@
     <Reference Include="System.Web" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <!--
+    This reference allows us to build the code without precompiler directives,
+    but the logic at runtime will never try to use the Registry if it's not available
+    -->
+    <PackageReference Include="Microsoft.Win32.Registry" Version="4.6.0" PrivateAssets="all" />
```

Comment:
When targeting `net45` and `net461`, is `Microsoft.Win32.RegistryKey.OpenBaseKey` already in mscorlib? I want to verify why we don't need to add a new reference when targeting those frameworks.

Reply:
Correct! In .NET Framework [it lives in `mscorlib`](https://apisof.net/catalog/Microsoft.Win32.RegistryKey.OpenBaseKey(RegistryHive,RegistryView)). The `RegistryKey` class since 1.1 and the `OpenBaseKey()` method since 4.0.

We don't need to include this as a transitive dependency (`PrivateAssets="all"`) because we never call into it from .NET Core.